provisioning/provisioning-device-client/src/main/java/com/microsoft/azure/sdk/iot/provisioning/device/internal/parser/DeviceRegistrationParser.java

```diff
@@ -30,7 +31,9 @@
      */
     static class TpmAttestation
     {
+        @SuppressWarnings("FieldCanBeLocal")
```

Comment:
Missing a comment here and on the other suppresions in this file

Reply:
I added comment for the ones which were not self explanatory